src/main/webapp/styles.css

```diff
@@ -0,0 +1,23 @@
+.flex-container {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.flex-item {
+  border: 1px solid black;
+  height: 500px;
```

Comment:
Using hardcoded values (numbers, decimals ... words like 'center' are fine) can actually be avoided in CSS. I recommend creating a new file (perhaps `variables.css`) and including your variable values in there. For Example:

```
/*
  In a variable file
*/
::root {
  --variable-name: variable value;
}

/*
  In another file
*/
.a-sample-class-name {
  property: var(--variable-name);
}
```

Reply:
This is a great idea, thanks for the suggestion!

Reply:
https://codeburst.io/css-variables-explained-with-5-examples-84adaffaa5bd If you need some more guidance!

Reply:
I managed to change most of the hard-coded values into variables except for `max-width` in `@media`. Seems like we would need a preprocessor for it so it might be best to retain that one hard-coded value. Here's an [article](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries) from StackOverflow that touches on this.

Reply:
`RESOLVED`